### DISCUSSION TITLE
Layout keine Anzeige

### DISCUSSION THREAD

**Discussion by sigi2345** [Zefau/ioBroker.jarvis#949] [General]
Hallo,

nach einen update auf 3.0.65 ist das Layout weg?

![Screenshot (4264)_LI](https://user-images.githubusercontent.com/41079106/132170145-a10d083d-2e32-4492-9779-4d8fbc8185b0.jpg)

**Comment by Zefau** [comment]
Danke für die Fehlermeldung. Behebe ich direkt.

**Reply by sigi2345** [reply]
Ok, hab leider auch eine issues gemacht.
https://github.com/Zefau/ioBroker.jarvis/issues/950

**Reply by Zefau** [reply]
Issue ist richtig, da eindeutig ein Fehler

**Reply by sigi2345** [reply]
Danke , funktioniert wieder.
